### ISSUE TITLE
wall not working on loklak.net

### ISSUE THREAD

**Issue by Orbiter** [loklak/loklak_webclient#477]
After I moved test.loklak.net to loklak.net, the wall is not working any more. The javascript terminal shows the following error:
```
"Error: [$rootScope:inprog] $apply already in progress
http://errors.angularjs.org/1.3.16/$rootScope/inprog?p0=%24apply
i/<@http://loklak.net/js/main.js:30:1177
f@http://loklak.net/js/main.js:31:30933
Ht/this.$get</h.prototype.$apply@http://loklak.net/js/main.js:33:2968
["/home/loklaktest/loklak_webclient/node_modules/angular/angular.js"]</</</or[t]</<.compile/</<@http://loklak.net/js/main.js:35:18841
jQuery.event.dispatch@http://code.jquery.com/jquery-2.1.4.js:4434:15
jQuery.event.add/elemData.handle@http://code.jquery.com/jquery-2.1.4.js:4121:6
jQuery.event.trigger@http://code.jquery.com/jquery-2.1.4.js:4350:5
.trigger/<@http://code.jquery.com/jquery-2.1.4.js:4901:4
.each@http://code.jquery.com/jquery-2.1.4.js:374:14
jQuery.prototype.each@http://code.jquery.com/jquery-2.1.4.js:139:10
.trigger@http://code.jquery.com/jquery-2.1.4.js:4900:1
i/e.proceed@http://loklak.net/js/main.js:22:16928
["/home/loklaktest/loklak_webclient/node_modules/angular/angular.js"]</</va.prototype.functionCall/<@http://loklak.net/js/main.js:35:12265
["/home/loklaktest/loklak_webclient/node_modules/angular/angular.js"]</</</or[t]</<.compile/</</a@http://loklak.net/js/main.js:35:18791
Ht/this.$get</h.prototype.$eval@http://loklak.net/js/main.js:33:2755
Ht/this.$get</h.prototype.$apply@http://loklak.net/js/main.js:33:2980
["/home/loklaktest/loklak_webclient/node_modules/angular/angular.js"]</</</or[t]</<.compile/</<@http://loklak.net/js/main.js:35:18841
jQuery.event.dispatch@http://code.jquery.com/jquery-2.1.4.js:4434:15
jQuery.event.add/elemData.handle@http://code.jquery.com/jquery-2.1.4.js:4121:6
"
```

**Comment by smokingwheels** [comment]
What about permissions, just a though.
http://stackoverflow.com/questions/18344551/move-file-as-root-preserving-ownership-linux

**Comment by aneeshd16** [comment]
This was caused by multiple problems:
1. The `domain` field in the config file was not updated to reflect the domain change.
2. oauth-proxy had to be restarted to load the new settings
3. The wall display was hidden due to a faulty `ng-show` in the body tag of index.html.

Now fixed: http://loklak.net/aneeshd16/wall/4yrbG0i5
Solved with d513cecb6003a026a748b1138e4145aac24e9831
